scripts/set-performance

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+source env-prep
+PERSIST=${PERSIST:-"transient"}
+oc exec -n $LOGGING_NS -c elasticsearch $pod -- es_util --query=_cluster/settings -XPUT -d "{\"${PERSIST}\":{\"cluster.routing.allocation.node_concurrent_incoming_recoveries\":\"50\"}}"
+
+oc exec -n $LOGGING_NS -c elasticsearch $pod -- es_util --query=_cluster/settings -XPUT -d "{\"${PERSIST}\":{\"cluster.routing.allocation.cluster_concurrent_rebalance\":\"50\"}}"
+
+oc exec -n $LOGGING_NS -c elasticsearch $pod -- es_util --query=_cluster/settings -XPUT -d "{\"${PERSIST}\":{\"cluster.routing.allocation.node_concurrent_outgoing_recoveries\":\"50\"}}"
+
+oc exec -n $LOGGING_NS -c elasticsearch $pod -- es_util --query=_cluster/settings -XPUT -d "{\"${PERSIST}\":{\"cluster.routing.allocation.node_concurrent_recoveries\":\"50\"}}"
+
+oc exec -n $LOGGING_NS -c elasticsearch $pod -- es_util --query=_cluster/settings -XPUT -d "{\"${PERSIST}\":{\"cluster.routing.allocation.node_initial_primaries_recoveries\":\"15\"}}"
+
+oc exec -n $LOGGING_NS -c elasticsearch $pod -- es_util --query=_cluster/settings -XPUT -d "{\"${PERSIST}\":{\"indices.recovery.max_bytes_per_sec\":\"150mb\"}}"
```

Comment:
Instead of 6 different oc commands, can we make one JSON document with all the settings and then apply it once?  This avoids having to get all the masters and nodes to update the settings globally across the cluster each time.

Reply:
@drewandersonnz, @jcantrill, any thoughts on this?